3Dadvect.py:174: RuntimeWarning: divide by zero encountered in double_scalars
  enst = ss.var('enst').sum()/enst0
3Dadvect.py:174: RuntimeWarning: divide by zero encountered in double_scalars
  enst = ss.var('enst').sum()/enst0
3Dadvect.py:174: RuntimeWarning: divide by zero encountered in double_scalars
  enst = ss.var('enst').sum()/enst0
3Dadvect.py:174: RuntimeWarning: divide by zero encountered in double_scalars
  enst = ss.var('enst').sum()/enst0
3Dadvect.py:174: RuntimeWarning: divide by zero encountered in double_scalars
  enst = ss.var('enst').sum()/enst0
3Dadvect.py:174: RuntimeWarning: divide by zero encountered in double_scalars
  enst = ss.var('enst').sum()/enst0
3Dadvect.py:174: RuntimeWarning: divide by zero encountered in double_scalars
  enst = ss.var('enst').sum()/enst0
3Dadvect.py:174: RuntimeWarning: divide by zero encountered in double_scalars
  enst = ss.var('enst').sum()/enst0
[ip-172-31-3-239:11585] 7 more processes have sent help message help-mpi-btl-base.txt / btl:no-nics
[ip-172-31-3-239:11585] Set MCA parameter "orte_base_help_aggregate" to 0 to see all help / error messages
